lua/neotest/consumers/output.lua

```diff
@@ -7,7 +7,7 @@ local win, short_opened
 local function open_output(result, opts)
   local buf = async.api.nvim_create_buf(false, true)
   local chan = async.api.nvim_open_term(buf, {})
-  local output = opts.short and result.short or lib.files.read(result.output)
+  local output = opts.short and result.short or (result.output and lib.files.read(result.output))
```

Comment:
This will still error on line 13 if there is no output.
Please change to this
```lua
  local output = opts.short and result.short or (result.output and lib.files.read(result.output))
  if not output then
    lib.notify("Output not found for position", "warn")
    return
  end
  local buf = async.api.nvim_create_buf(false, true)
  local chan = async.api.nvim_open_term(buf, {})
  ...
```

Reply:
I added that check (which is a good one), but I now get that notification every time the test has an error (which is less impactful than the original traceback)
It seems that `open_output` gets called _three_ times - the first time `result` doesn't have the `output` field:
```
{
  errors = { {
      message = "ZeroDivisionError: division by zero"
    } },
  short = "\27[31m\27[1m______________________________________________________ test_init _____________________________
__________________________\27[0m\n\n <SNIP>",
  status = "failed"
}
```
then the second two invocations are identical, with the addition of the `output` field:
```
{
  errors = { {
      message = "ZeroDivisionError: division by zero"
    } },
  output = "/tmp/nvim.josh/Kx4O2K/3",
  short = "\27[31m\27[1m______________________________________________________ test_init _____________________________
__________________________\27[0m\n\n <SNIP>",
  status = "failed"
}
```

Reply:
Looks like it is because of the `results_callback` begin called at least twice - once for each stream result and then on the final result:
  https://github.com/nvim-neotest/neotest/blob/master/lua/neotest/client/runner.lua#L89-L99